specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -258,7 +261,135 @@ sealed partial class App : Application
   }
 }
 ```
+## Registration of the Push Activator for unpackaged apps for protocol activation with PushNotificationsLongRunningTask
+The main difference here for unpackaged applications is that RegisterActivator will consume ProtocolActivator</br>
+and register with PushNotificationsLongRunningTask, a long running task that is shipped with PRmain, to handle</br>
+receiving raw notifications.
+
+**NOTE:** Packaged applications running on Windows builds earlier than 19041 will need to use ProtocolActivator</br>
+due to SetTaskEntryPointClsid() being unavailable on those builds.
+
+```cpp
+int main()
+{
+    // Take a dependency on Windows App SDK preview 
+    const UINT32 majorMinorVersion{ 0x00000008 }; 
+    PCWSTR versionTag{ L"preview" }; 
+    const PACKAGE_VERSION minVersion{};
+
+    const HRESULT hr{ MddBootstrapInitialize(majorMinorVersion, versionTag, minVersion) }; 
+    // Check the return code. If there is a failure, display the result.
+    if (FAILED(hr)) 
+    { 
+        wprintf(L"Error 0x%X in MddBootstrapInitialize(0x%08X, %s, %hu.%hu.%hu.%hu)\n", 
+            hr, majorMinorVersion, versionTag, minVersion.Major, minVersion.Minor, minVersion.Build, minVersion.Revision); 
+        return hr; 
+    }
+
+    PushNotificationRegistrationOptions pushOptions = PushNotificationManager::GetPushActivatorOptions(true);
```

Comment:
For Unpackaged apps, what does it mean to set inproc to false? Does that mean that we launch a new process when the app is running?

Reply:
It would be nice to have that support. @danielayala94 How could we support out-of-proc activation in your COM design?